app/lib/meadow/notification/heartbeat.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Meadow.Notification.Heartbeat do
+  @moduledoc """
+  GenServer for sending periodic heartbeat notifications.
+  """
+
+  use GenServer
+  require Logger
+  alias Meadow.Notification
+
+  @default_interval :timer.seconds(5)
+
+  def start(message, info, interval \\ @default_interval) do
+    Logger.info("Starting heartbeat for #{inspect(info)} every #{interval} ms")
+    state = %{message: message, info: info, interval: interval}
+    GenServer.start_link(__MODULE__, state, name: __MODULE__)
```

Comment:
```suggestion
    GenServer.start_link(__MODULE__, state)
```

I think this is causing the heartbeat (and also the whole plan creation) not to work if multiple users/browsers send chat messages simultaneously and each tries to start a Heartbeat GenServer with the same name. I am getting a `{:error, {:already_started, #PID<0.10258.0>}}` error from the `sendChatMessage` graphQL mutation in the 2nd browser window.